HISTORY.rst

```diff
@@ -4,6 +4,14 @@ Release History
 6.0.0dev0
 ---------
 
+5.2.0 (2019-01-18)
+------------------
+
+**Other Changes**
```

Comment:
I think this should say `API Changes (Backward-compatible)`, as in previous releases - IFF it applies (?)

Reply:
I also wasn't sure this counted as an API change, I guess it is - will change.